src/___new___/components/VersionedCloudImageList/index.js

```diff
@@ -0,0 +1,61 @@
+import React, { useState } from 'react';
+import { Box, Select, Flex, FormControl, FormLabel } from '@chakra-ui/react';
+import CodeBlock from '@theme/CodeBlock';
+import variables from '../../../../calico-cloud/variables';
+import ccImageLists from '../../data/ccImageLists';
+
+const CodeBlockSelector = () => {
+  const [selectedVersion, setSelectedVersion] = useState(Object.keys(ccImageLists)[0]);
+
+  const handleSelectChange = (event) => {
+    setSelectedVersion(event.target.value);
+  };
+
+  return (
+    <Flex
+      gap={4}
+      flexDir='column'
+    >
+      <Box
+        border='1px solid'
+        borderColor='gray.300'
+        borderRadius='md'
+        p='4'
+        w='33%'
+      >
+        <FormControl>
+          <FormLabel>
+            <h5>Select your version:</h5>
+          </FormLabel>
+          <Select
+            width='100%'
+            value={selectedVersion}
+            onChange={handleSelectChange}
+          >
+            {Object.keys(ccImageLists).map((version) => (
+              <option
+                key={version}
+                value={version}
+              >
+                {version}
+              </option>
+            ))}
+          </Select>
+        </FormControl>
+      </Box>
+      <CodeBlock
+        title={`Helm command to add images for Calico Cloud ${selectedVersion}`}
+        language='javascript'
```

Comment:
this should match whatever languar is in the code block

Reply:
done